{"name": "CVE-2013-2104", "affected_release": {"advisory": "RHSA-2013:0944", "release_date": "2013-06-12T00:00:00", "cpe": "cpe:/a:redhat:openstack:3::el6", "product_name": "Red Hat Enterprise Linux OpenStack Platform 3.0", "package": "python-keystoneclient-1:0.2.3-2.el6ost"}, "public_date": "2013-05-28T00:00:00", "package_state": {"fix_state": "Affected", "product_name": "Red Hat OpenStack Platform 2.1", "package_name": "openstack-keystone", "cpe": "cpe:/a:redhat:openstack:2.1"}, "acknowledgement": "\nThis issue was discovered by Eoghan Glynn of Red Hat.\n    ", "document_distribution": "Copyright \u00a9 2016 Red Hat, Inc. All rights reserved.", "threat_severity": "Moderate", "bugzilla": {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=965852", "description": "\nCVE-2013-2104 OpenStack Keystone: Missing expiration check in Keystone PKI token validation\n    ", "id": "965852"}, "details": ["\npython-keystoneclient before 0.2.4, as used in OpenStack Keystone (Folsom), does not properly check expiry for PKI tokens, which allows remote authenticated users to (1) retain use of a token after it has expired, or (2) use a revoked token once it expires.\n    "], "cwe": "CWE-613", "cvss": {"cvss_scoring_vector": "AV:N/AC:H/Au:N/C:N/I:P/A:N", "status": "verified", "cvss_base_score": "2.6"}}